Sources/StreamChat/WebSocketClient/Events/NotificationEvents.swift

```diff
@@ -149,3 +149,16 @@ public struct NotificationInviteRejectedEvent<ExtraData: ExtraDataTypes>: EventW
         payload = response
     }
 }
+
+public struct NotificationChannelMutesUpdatedEvent: EventWithUserPayload {
+    public let userId: UserId
+    let payload: Any
+    
+    init<ExtraData: ExtraDataTypes>(from response: EventPayload<ExtraData>) throws {
+        guard try response.value(at: \.memberContainer?.invite?.isInvited) == true else {
```

Comment:
I just realized that the  payload will never contain `memberContainer`, it's probably safe to delete this code.